### PR TITLE
Recipe for integrate tagria lib with vraptor 

### PR DESCRIPTION
**Pull request by angeliski**
Hey @Turini and @jslsolucoes, I maked the changes. Can you review?

### CODE REVIEW

File: vraptor-site/layouts/en/cookbook.html [hunk 1]
```diff
@@ -41,6 +41,11 @@
               Using AngularJS on VRaptor4
             </a>
         </li>
+        <li>
+            <a href="/en/cookbook/integrating-vraptor-with-tagrialib">
+              Integrating VRaptor with TagriaLib
```
Comment (jslsolucoes) [inline vraptor-site/layouts/en/cookbook.html]: Please , change TagriaLib to Tagria

File: vraptor-site/content/pt/cookbook/integrando-vraptor-com-tagrialib.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: Integrando VRaptor com Tagria
+---
+
+# Integrando VRaptor com Tagria
+
+## por Jonatan Lemes
+
+Tagria é uma suite open source de componentes que te ajuda a construir lindas view em minutos.
+Tagria integra perfeitamente com qualquer framework MVC baseado em ação incluindo Vraptor
```
Comment (jslsolucoes) [inline vraptor-site/content/pt/cookbook/integrando-vraptor-com-tagrialib.html]: Please , change Vraptor to VRaptor

File: vraptor-site/content/en/cookbook/integrating-vraptor-with-tagrialib.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
+title: Integrating VRaptor with Tagria
+---
+
+# Integrating VRaptor with Tagria
+
+## by Jonatan Lemes
+
+Tagria is an open source component suit framework that help build beautiful view in minutes. 
+Tagria integrates perfectly with any action based mvc framework for java including VRaptor.
```
Comment (jslsolucoes) [inline vraptor-site/content/en/cookbook/integrating-vraptor-with-tagrialib.html]: Please , change mvc to MVC (uppercase)

File: vraptor-site/content/en/cookbook/integrating-vraptor-with-tagrialib.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+You can found more information visiting <a href="https://github.com/jslsolucoes/tagria">Tagria project</a>.
+
+The first step is include the Tagria dependency in your Maven project as described below. 
+
+~~~
+#!xml
+<dependency>
+	<groupId>com.jslsolucoes</groupId>
+	<artifactId>tagria-lib</artifactId>
+	<version>1.0.1</version> <!-- or the latest version -->
```
Comment (jslsolucoes) [inline vraptor-site/content/en/cookbook/integrating-vraptor-with-tagrialib.html]: Please if you can update version to new stable 1.0.2

**Comment by jslsolucoes** [comment]
@angeliski Thanks for the new pull request and some points was added for review. After that sounds ok to me.

**Comment by Turini** [comment]
sounds great to me. 
we can merge/release it just after these minors are done. 
thank you guys

**Comment by angeliski** [comment]
Done

**Comment by jslsolucoes** [comment]
It`s ok by now . Thanks again @angeliski

**Comment by Turini** [comment]
merged. thank you guys

**Pull request merged**